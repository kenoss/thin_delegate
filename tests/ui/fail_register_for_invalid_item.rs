#[thin_delegate::register]
fn hoge() -> bool {
    true
}

fn main() {}
