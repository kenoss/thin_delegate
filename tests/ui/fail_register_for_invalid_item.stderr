error: expected `trait ...` or `struct ...` or `enum ...`
 --> tests/ui/fail_register_for_invalid_item.rs:2:1
  |
2 | / fn hoge() -> bool {
3 | |     true
4 | | }
  | |_^
